Kudu.Core/Helpers/EnvironmentHelper.cs

```diff
@@ -27,18 +27,6 @@ public static string NormalizeBinPath(string binPath)
         // Is this a Windows Containers site?
         public static bool IsWindowsContainers()
         {
-            // OLD WAY: This can be removed after ANT84
```

Comment:
Saw this old code in here - can be removed now?

Reply:
Please remove if no upstream usage.  thank.

Reply:
Joaquin added this originally - I'll confirm with him that this can be removed

Reply:
Yes, that can be removed

---
In reply to: [331590375](https://github.com/projectkudu/kudu/pull/3049#discussion_r331590375) [](ancestors = 331590375)